classes/npm.bbclass

```diff
@@ -3,7 +3,8 @@
 # SPDX-License-Identifier: MIT
 
 # HOWTO generate an npm-shrinkwrap.json:
-#   npm install --global-style <my-favorite-package>
+#   npm config set global-style=true (you may want to reset this afterward)
```

Comment:
I used -- global style to generate node red based on NPM 7.5.2
Then the shrink file is exported and tested, which seems to be correct

Reply:
As `--global-style` was swallowed but had on effect, you created a flattened installation, and all dependencies landed outside of the node-red module (as it is the default). But that cause the packaging to ignore them, and Node RED was missing them on the target.